pkgs/applications/editors/jetbrains/plugins/plugins.json

```diff
@@ -153,11 +153,10 @@
         "webstorm"
       ],
       "builds": {
-        "231.9011.34": "https://plugins.jetbrains.com/files/8554/326468/featuresTrainer-231.8770.66.zip",
-        "231.9011.35": "https://plugins.jetbrains.com/files/8554/326468/featuresTrainer-231.8770.66.zip",
-        "231.9011.38": "https://plugins.jetbrains.com/files/8554/326468/featuresTrainer-231.8770.66.zip",
-        "231.9011.41": "https://plugins.jetbrains.com/files/8554/326468/featuresTrainer-231.8770.66.zip",
-        "231.9161.38": "https://plugins.jetbrains.com/files/8554/326468/featuresTrainer-231.8770.66.zip"
+        "231.9161.29": "https://plugins.jetbrains.com/files/8554/326468/featuresTrainer-231.8770.66.zip",
+        "231.9161.38": "https://plugins.jetbrains.com/files/8554/326468/featuresTrainer-231.8770.66.zip",
+        "231.9161.40": "https://plugins.jetbrains.com/files/8554/326468/featuresTrainer-231.8770.66.zip",
+        "231.9161.41": "https://plugins.jetbrains.com/files/8554/326468/featuresTrainer-231.8770.66.zip"
```

Comment:
Here is one more deletion then additon is that because a plugin was removed?

Reply:
different ide builds support different plugin builds, which sometimes means you go from each ide having it's own version of a plugin to two ides supporting the same plugin, so the total amount of plugins supported drops by one